releases/release-6.1.2.md

```diff
@@ -0,0 +1,130 @@
+    - (dup) Fix the issue that the `EXECUTE` might throw an unexpected error in specific scenarios [#37187](https://github.com/pingcap/tidb/issues/37187)
+
++ TiKV **owner: @ethercflow**
+
+    - Fix the issue that snapshot might not contain the complete data caused by introduced batch snapshot across regions [#13553](https://github.com/tikv/tikv/issues/13553)
+    - (dup) Fix the issue of QPS drop when flow control is enabled and `level0_slowdown_trigger` is set explicitly [#11424](https://github.com/tikv/tikv/issues/11424)
+    - (dup) Fix the issue that causes permission denied error when TiKV gets an error from the web identity provider and fails back to the default provider [#13122](https://github.com/tikv/tikv/issues/13122)
+    - (dup) Fix the issue that the TiKV service is unavailable for several minutes when a TiKV instance is in an isolated network environment [#12966](https://github.com/tikv/tikv/issues/12966)
+
++ PD **owner: @nolouch**
+
+    - Fix the issue that the statistics of the region tree might be not accurate [#5318](https://github.com/tikv/pd/issues/5318)
+    - (dup) Fix the issue that the TiFlash learner replica might not be created [#5401](https://github.com/tikv/pd/issues/5401)
+    - (dup) Fix the issue that PD cannot correctly handle dashboard proxy requests [#5321](https://github.com/tikv/pd/issues/5321)
+    - (dup) Fix the issue that unhealthy Region might cause PD panic [#5491](https://github.com/tikv/pd/issues/5491)
+
++ TiFlash
+
+    <!--compute **owner: @zanmato1984**-->
+
+    - (dup) Fix the issue that a window function might cause TiFlash to crash when the query is canceled [#5814](https://github.com/pingcap/tiflash/issues/5814)
+
+    <!--storage **owner: @flowbehappy**-->
+
+    - Fix the issue that iolimiter could unexpectedly throttle io throughput of query requests after large volume writes, which slows down query performance [#5801](https://github.com/pingcap/tiflash/issues/5801)
+    - (dup) Fix the panic that occurs after creating the primary index with a column containing the `NULL` value [#5859](https://github.com/pingcap/tiflash/issues/5859)
+
++ Tools
+
+    + TiDB Lightning **owner: @niubell**
+
+    + TiDB Data Migration (DM) **owner: @niubell**
+
+        - DM will try to persist upstream table structure from dump files when firstly switch to sync unit [#5010](https://github.com/pingcap/tiflow/issues/5010)
+        - DM will try to persist upstream table structure from dump files when firstly switch to sync unit [#7159](https://github.com/pingcap/tiflow/issues/7159)
+        - DM precheck no longer reports lacking privileges of INFORMATION_SCHEMA [#7317](https://github.com/pingcap/tiflow/issues/7317)
+        - (dup) Fix the issue that DM reports the `Specified key was too long` error [#5315](https://github.com/pingcap/tiflow/issues/5315)
+        - (dup) Fix the issue that latin1 data might be corrupted during replication [#7028](https://github.com/pingcap/tiflow/issues/7028)
+
```

Comment:
Fix the issue dm-worker hangs after running dm tasks with fast/full validators [#7241](https://github.com/pingcap/tiflow/issues/7241)